### PR TITLE
Fix/326 no display mode change in nav view

### PR DESCRIPTION
**Pull request by YarikMamykin**
Implements/Fixes [#326](https://github.com/smartdevicelink/sdl_hmi/issues/326)

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
In case of display mode value change there was no logic for changing display mode in navigation view. Besides that mechanics of DisplayMode selection view was changed: since now this control element just changes value of `SDL.SDLModel.data.imageMode` property and all interested observers are notified each time it changes to process such behavior with appropriate callback.

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by YarikMamykin** [comment]
@theresalech please notice that this PR is ready for Livio Team Code Review.

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by jacobkeeler** [comment]
@YarikMamykin also fix merge conflicts

File: app/controller/sdl/Abstract/Controller.js [hunk 1]
```diff
@@ -1549,5 +1549,26 @@ SDL.SDLController = Em.Object.extend(
...
+      }
+
+      SDL.SDLController.model.setMode(SDL.SDLModel.data.imageMode);
+      var commands = SDL.SDLController.model.get('currentCommandsList');
+      const length = SDL.OptionsView.commands.items.length;
+      for(var i=0;i<length;i++){
+        SDL.OptionsView.commands.items[i].type.prototype.setMode(SDL.SDLModel.data.imageMode);
+        if(commands[i].isTemplate){
+          SDL.OptionsView.commands.items[i].type.prototype.setMode(SDL.SDLModel.data.imageMode);
+        }
```
[the remark this answers is not in the document]
Comment (AKalinich-Luxoft) [reply app/controller/sdl/Abstract/Controller.js]: @jacobkeeler redundant code removed in https://github.com/smartdevicelink/sdl_hmi/pull/369/commits/abe9cc55b9b3e890a48ce49e75f88bed695b553c

**Comment by AKalinich-Luxoft** [comment]
> @YarikMamykin also fix merge conflicts

@jacobkeeler conflicts have been resolved in https://github.com/smartdevicelink/sdl_hmi/pull/369/commits/8e27920fb936f41b885d8eb4598d8e04929609c3

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**